example/C/src/Rhythm/SensorSimulator.lf

```diff
@@ -5,24 +5,27 @@
 target C {
     threads: 2,
     flags: "-lncurses",
+    cmake: false,
     files: ["/lib/C/util/sensor_simulator.c", "/lib/C/util/sensor_simulator.h"]
 };
 preamble {=
     #include "sensor_simulator.c"
```

Comment:
Why is the `.c` file included, not the `.h` file?

Reply:
Because the .c file includes the .h file.

Reply:
That's not how header files are supposed to be used....

Reply:
Right. Now that we are using CMake, I presume that some song and dance can be used instead so that the .c file also gets compiled and linked.